lib/index.js

```diff
@@ -42,11 +42,19 @@ var http = require('http');
 var utils = require('./utils');
 var winston = require('winston');
 
-function AppEngine() {
+function AppEngine(options) {
+  this.options_ = options || {};
   this.serializer_ = new net.proto2.contrib.WireSerializer();
   this.cache_ = {};
-  this.logSizeThreshold_ = 1024 * 1024; // 1 MB
-  this.logMaxTimestampDeltaMillis_ = 60000; // 60 seconds
+  this.defaults_ = {
```

Comment:
Did you mean "remove from" or "move to"? E.g.
AppEngine.prototype.defaultOptions_ = { foo: 1, bar: 2 };